lib/_stream_writable.js

```diff
@@ -293,22 +293,16 @@ function doWrite(stream, state, writev, len, chunk, encoding, cb) {
 }
 
 function onwriteError(stream, state, sync, er, cb) {
+  --state.pendingcb;
```

Comment:
I'm confused how this isn't a change functionality? It went from decrementing only in the `else` to always.